tg/models.py

```diff
@@ -11,7 +11,7 @@
 
 
 class Model:
-    def __init__(self, tg: Tdlib) -> None:
```

Comment:
why?  we need to be consistent and either remove all constructor annotations  at once or  leave it as is :slightly_smiling_face: